### PR TITLE
Fix Octicons-related issues

### PR DESCRIPTION
**Pull request by fregante**
I'll push here any improvements related to https://github.com/sindresorhus/refined-github/pull/3147

Feel free to contribute your own PRs

### CODE REVIEW

File: source/refined-github.css [hunk 1]
```diff
@@ -116,3 +116,8 @@ pr-branches
 .js-issue-row .text-small.text-gray {
 	line-height: 1.8;
 }
+
+/* Reset `vertical-align` for .octicons to work with above increased line-height */
+.js-issue-row .text-small.text-gray .octicon {
```
Comment (yakov116) [inline source/refined-github.css:121]:
```suggestion
.rgh-align-issue-labels .js-issue-row .text-small.text-gray .octicon {
```
With `rgh-align-issue-labels` turned off the alignment is a drop off

Comment (fregante) [reply source/refined-github.css:121]: This is unrelated to that feature, the spacing changes regardless of that

Comment (fspoettel) [reply source/refined-github.css:121]: @yakov116 The highlighted change is concerned with the alignment of the `clock` icon which looks correct with `align-labels` turned off:

Screenshot from 2020-05-28 13-51-21

Comment (yakov116) [reply source/refined-github.css:121]: Your both right, I thought i noticed  a small change.

**Pull request merged**